### PR TITLE
TCVP-1268 fixed docker-compose to include minio

### PR DESCRIPTION
**Pull request by KDNeufeld**
# Description

This PR includes the following proposed change(s):

TCVP-1268 
- fixed an issue with the env vars in docker-compose to properly wire minio with staff-api
- added env vars to staff-api helm

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

docker-compose up 
Tested that staff-api works properly

## Does the change impact or break the Docker build?

- [x] Yes
- [ ] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .gitops/charts/traffic-court-online/charts/staff-api/templates/deployment.yaml [hunk 1]
```diff
@@ -53,6 +53,17 @@ spec:
             value: "{{ .Values.rabbitmq.username }}"
           - name: RabbitMQ__Password
             value: "{{ .Values.rabbitmq.password }}"
+          - name: TicketStorage__Type
```
Comment (pbolduc) [inline .gitops/charts/traffic-court-online/charts/staff-api/templates/deployment.yaml:56]: This defaults to object storage if not defined. So I prefer to skip it. The ObjectStorage parameters below are not conditional.

File: .gitops/charts/traffic-court-online/charts/staff-api/values.yaml [hunk 2]
```diff
@@ -116,6 +116,18 @@ jaeger:
...
 
+ticketStorage:
+  # options are ObjectStore or InMemory
+  type:
+  
+objectStorage:
+  endpoint: 
+  bucketName: 
+  # ie. username
+  accessKey:
```
Comment (pbolduc) [inline .gitops/charts/traffic-court-online/charts/staff-api/values.yaml:127]: Please put these as comments above 
```
## @param
## objectStorage.accessKey The username used to access the bucket
## objectStorage.secretKey The password used to access the bucket
```

File: .gitops/charts/traffic-court-online/charts/staff-api/templates/deployment.yaml [hunk 3]
```diff
@@ -53,6 +53,17 @@ spec:
...
+            value: "{{ .Values.ticketStorage.type }}"
+          # Object Storage
+          - name: ObjectStorage__AccessKey
+            value: "{{ .Values.objectStorage.accessKey }}"
```
Comment (pbolduc) [inline .gitops/charts/traffic-court-online/charts/staff-api/templates/deployment.yaml:60]: If these are required, add required check

```
{{ required "A valid .Values.objectStorage.accessKey required!" .Values.objectStorage.accessKey }}
```

**Review by pbolduc — CHANGES_REQUESTED** [review]

File: .gitops/charts/traffic-court-online/charts/staff-api/templates/deployment.yaml (hunk 1 above)
Comment (KDNeufeld) [reply .gitops/charts/traffic-court-online/charts/staff-api/templates/deployment.yaml]: Done, removed. Left to defaults

File: .gitops/charts/traffic-court-online/charts/staff-api/templates/deployment.yaml (hunk 3 above)
Comment (KDNeufeld) [reply .gitops/charts/traffic-court-online/charts/staff-api/templates/deployment.yaml]: Done.

File: .gitops/charts/traffic-court-online/charts/staff-api/values.yaml (hunk 2 above)
Comment (KDNeufeld) [reply .gitops/charts/traffic-court-online/charts/staff-api/values.yaml:130]: Done

**Review by pbolduc — APPROVED** [review]

**Pull request merged**